tripal/tripal_jbrowse
# 35 multiple jbrowses 41organism

Include analysis(analysis_id) to have multiple JBrowse instances for one organism.

How to test:
1. since database(table: tripal_jbrowse_mgmt_instances) of for instances needs one more column of analysis_id, so you may need to run update to update the module
**WARNING: you will lose all instances created if the module is re-installed**
2. select one organism which needs multiple instances (like have different genome versions), create each one with specific analysis
3. in general, test if tripal_jbrowse still works as expected by creating instance, managing tracks
